.buildkite/post-commit.yml

```diff
@@ -5,13 +5,6 @@ env:
   # Per-host variables - shared across containers on host
   CACHE_DIR: "/cache/cardano-wallet"
 
-steps:
-env:
-  LC_ALL: "en_US.UTF-8"
-  NIX_PATH: "channel:nixos-21.11"
```

Comment:
Do you believe these are redundant or is there are more specific reason for dropping?

Reply:
no idea of the value here, we can remove and see if it works